cmapPy/math/robust_zscore.py

```diff
@@ -0,0 +1,31 @@
+'''
+robust_zscore.py
+
+Given a pandas df, and an optional control df, will calculate zscores using plate control or vehicle control
```

Comment:
Again, since this is a public repo you need to define all these terms more precisely. 

- What's an optional control df? 
- (Briefly); what's the difference between plate control or vehicle control? It would be ideally good to point to connectopedia or something for this, but I don't see it as an article there; consider coordinating with Sarah/Jenny to add it there instead?